Formula/picotool.rb

```diff
@@ -0,0 +1,30 @@
+class Picotool < Formula
+  desc "RP2040 BOOTSEL and binary utility"
```

Comment:
From its [help text](https://github.com/raspberrypi/picotool#overview):

```suggestion
  desc "Tool for interacting with a RP2040 device in BOOTSEL mode, or with a RP2040 binary"
```